README.md

```diff
@@ -24,6 +24,31 @@
 * Mapstruct
 * Lombok
 
+## Start application with docker
```

Comment:
1. Please rename is to Start the application using docker
2. This section does not contain instructions about how to start your application using docker.
After a short description of the Dockerfile and docker-compose.yml file (you already have it), you should add the steps to run the app using docker. E.g.

- download the app
$ git clone blabla
- create a jar file
$ mvn clean package
- run the app in the docker
$ docker-compose up
and so on
The last command should be
- test the app is up and running
$ curl --location --request GET 'https://here-should-go-the-request-that-retrieves-all-countries?'

Reply:
done